build.zig

```diff
@@ -98,11 +99,7 @@ pub fn build(b: *Builder) !void {
 
     const test_stage2_step = b.step("test-stage2", "Run the stage2 compiler tests");
     test_stage2_step.dependOn(&test_stage2.step);
-
-    // TODO see https://github.com/ziglang/zig/issues/1364
-    if (false) {
-        test_step.dependOn(test_stage2_step);
-    }
+    test_step.dependOn(test_stage2_step);
```

Comment:
I don't think the stage2 tests are ready to be resurrected yet, if you look at `src-self-hosted/test.zig`, it creates its own event loop, and tries to do evented I/O, but `zig test` doesn't support that yet, so it's this weird mixture of evented and blocking (which amazingly apparently did work successfully on Linux and macOS)